extensions/sequelize/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@loopback/sequelize",
+  "version": "0.0.1",
+  "description": "@loopback/sequelize",
+  "keywords": [
+    "loopback-extension",
+    "loopback",
+    "sequelize-connector",
+    "sequelize"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": "14 || 16 || 17 || 18"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "pretest": "npm run rebuild",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "clean": "lb-clean dist *.tsbuildinfo .eslintcache",
+    "rebuild": "npm run clean && npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/loopbackio/loopback-next.git",
+    "directory": "extensions/sequelize"
+  },
+  "author": "IBM Corp. and LoopBack contributors",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "peerDependencies": {
+    "@loopback/core": "^4.0.4",
+    "@loopback/repository": "^5.0.4"
+  },
+  "dependencies": {
+    "debug": "^4.3.4",
+    "sequelize": "^6.25.3",
+    "tslib": "^2.0.0"
+  },
+  "devDependencies": {
+    "@loopback/boot": "^5.0.4",
+    "@loopback/build": "^9.0.4",
+    "@loopback/core": "^4.0.4",
+    "@loopback/repository": "^5.1.0",
+    "@loopback/rest": "^12.0.4",
+    "@loopback/testlab": "^5.0.4",
+    "@types/node": "^14.18.32",
+    "loopback-connector-sqlite3": "^3.0.0",
```

Comment:
This should not be necessary: Sequelize and `loopback-connector-*` are not interopable with each other.

Reply:
Sure I'll remove that. But could you please look into [this problem while finding sqlite3](https://github.com/loopbackio/loopback-next/pull/9048#issuecomment-1332366745)

The first line in [stack trace](https://github.com/loopbackio/loopback-next/actions/runs/3572042631/jobs/6004489041) clearly hints about it. That it try to find it on packages/repository level instead of the actual extension.